### ISSUE TITLE
How to specify the value of noise?

### ISSUE THREAD

**Issue by kangiscoding** [villoren/KalmanLocationManager#5]
Hi, thank you for sharing this wonderful work! I am new here. And I am still confused that how you specify the value of noise such as COORDINATE_NOISE and ALTITUDE_NOISE etc. Thank you.

**Comment by brainahdev** [comment]
Yes I too have this confusion and it will be better if any specification available

**Comment by habeebrahmanpt** [comment]
Yes I too have this confusion
